src/tactic/group.lean

```diff
@@ -0,0 +1,105 @@
+/-
+Copyright (c) 2020. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author: Patrick Massot
+-/
+import tactic
+import tactic.doc_commands
+
+/-!
+# `group`
+
+Normalizes expressions in the language of groups.
+
+## Tags
+
+group_theory
+-/
+
+-- The next four lemmas are not general purpose lemmas, they are intended for use only by
+-- the `group` tactic.
```

Comment:
Maybe put these lemmas inside a `tactic.group` namespace, just to keep them out of the way?

Reply:
Good idea.